src/main/java/beans/menus/EntityListResponse.java

```diff
@@ -230,7 +230,13 @@ private void processStyles(Detail detail) {
     }
 
     private void processActions(SessionWrapper session) {
-        Vector<Action> actions = session.getDetail(((EntityDatum) session.getNeededDatum()).getShortDetail()).getCustomActions(session.getEvaluationContext());
+        EntityDatum datum = (EntityDatum) session.getNeededDatum();
+        // HACKY! Uses the fact that case claim results will always be in a "result" instance
```

Comment:
I think this is actually defined in the `storage-instance` of the query node, so if this is the route you take, this should actually take the name that is defined there. 
https://github.com/dimagi/commcare-core/wiki/Suite20#remote-request